libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/offboard/MapboxOffboardRouter.kt

```diff
@@ -50,7 +51,7 @@ class MapboxOffboardRouter(
     ) {
         mapboxDirections = RouteBuilderProvider
             .getBuilder(accessToken, context, urlSkuTokenProvider)
-            .routeOptions(routeOptions)
+            .routeOptions(routeOptions, refreshEnabled)
```

Comment:
I didn't follow why we change the flow of enabling refresh?

Reply:
Updated the original post.